grpc-api/src/main/proto/geese.proto

```diff
@@ -47,6 +47,16 @@ message GeeseResponse {
     repeated GeeseLine lines = 1;
 }
 
+/*
+ * Geese Types
+ */
+enum GooseType {
+    CANADA_GOOSE = 0;
```

Comment:
By convention, 0 index element of enum should be `UNSPECIFIED`: `GOOSE_TYPE_UNSPECIFIED`

Reply:
Right now, the default and only goose type on the frontend is a Canada goose. By making it a default value - we will ensure compatibility between the new and the old API.

Reply:
No, 0 should be UNSPECIFIED which is different from default: https://developers.google.com/protocol-buffers/docs/style#enums

Frontend can fallback to whatever value it wants, and semantics of unspecified value shouldn't depend on historical values.

Reply:
Fair enough, will address.

@HalloGene please take re. the frontend fallback values.

Reply:
Fixed